src/mongo/db/storage/storage_engine_init.cpp

```diff
@@ -64,6 +67,34 @@ namespace {
  * as appropriate.
  */
 void createLockFile(ServiceContext* service);
+
+void writeMetadata(std::unique_ptr<StorageEngineMetadata> metadata,
+                   const StorageEngine::Factory* factory,
+                   const StorageGlobalParams& params,
+                   const KmipKeyIdPair& kmipKeyIds,
```

Comment:
There is no need to pass the whole `kmipKeyIds` object because the function uses only `kmipKeyIds.encryption`.